scripts/apps/ingest/directives/IngestSourcesContent.js

```diff
@@ -505,6 +505,9 @@ export function IngestSourcesContent(ingestSources, gettext, notify, api, $locat
                  * @param {string} fileType
                  */
                 $scope.addOrRemoveFileType = function(fileType) {
+                    let modalBodyContainer = angular.element('.modal__body');
```

Comment:
Don't query the DOM. Pass the form as a parameter instead.

Reply:
done